examples/acquiring_data/downloading_cutouts.py

```diff
@@ -38,6 +38,9 @@
 # Exporting data from the JSOC requires registering your
 # email first. Please replace this with your email
 # address once you have registered.
+# use "export JSOC_EMAIL='your_email@example.com'"
+# in your terminal to set the environment variable.
+# NO CHANGE REQUIRED HERE!
```

Comment:
This should not be what we tell users to change.

They should just change the line to be `jsoc_email="<their_email_address"` and not to export the variable.